docker-compose.yml

```diff
@@ -1,21 +1,12 @@
-version: '3'
+version: "3"
 services:
-  postgres:
-    build:
-      context: .
-      dockerfile: docker/Dockerfile.postgres
-    restart: unless-stopped
-    volumes:
-      - postgresdata:/var/lib/postgresql/data
   invidious:
     build:
       context: .
       dockerfile: docker/Dockerfile
     restart: unless-stopped
     ports:
       - "127.0.0.1:3000:3000"
-    depends_on:
-      - postgres
 
 volumes:
   postgresdata:
```

Comment:
This should also be removed, otherwise it will create a default network, etc. for unneeded `postgresdata`.